vm/package/deploy.rs

```diff
@@ -158,23 +179,21 @@ impl<N: Network> Package<N> {
         // Initialize the RNG.
         let rng = &mut rand::thread_rng();
         // Compute the deployment.
-        let deployment = process.deploy::<A, _>(program, rng).unwrap();
+        let deployment = process.deploy::<A, _>(program, rng)?;
+        // Compute the additional fee.
+        let (_, additional_fee) = process.execute_additional_fee::<A, _>(private_key, credits, 1, rng)?;
```

Comment:
Can we pass in an `additional_fee: u64` instead of hardcoding `1`?

Reply:
I'll do the same for #1140.